pkg/proxy/util/nfacct/nfacct_linux.go

```diff
@@ -160,7 +160,7 @@ func (r *runner) List() ([]*Counter, error) {
 		}
 		counters = append(counters, counter)
 	}
-	return counters, nil
+	return counters, err
```

Comment:
You should update the `nfacct.Interface` docs to mention that it may return a partial list with an error.